### PR TITLE
docs: update cli docs for exclusive topics

### PR DESCRIPTION
**Pull request by lafirest**
docs for https://github.com/emqx/emqx/pull/13524

### CODE REVIEW

**Comment by Meggielqk** [comment]
Is the CLI named "exclusive" or "exclusive topics"?

**Comment by lafirest** [comment]
> Is the CLI named "exclusive" or "exclusive topics"?

`exclusive`

File: en_US/admin/cli.md [hunk 1]
```diff
@@ -339,6 +339,29 @@ t/1 -> emqx@127.0.0.1
 If there are a large number of topic subscriptions in the cluster, the `list` command may be time-consuming and resource-intensive.
 :::
 
+## exclusive topics
```
Comment (Meggielqk) [inline en_US/admin/cli.md]:
```suggestion
## exclusive
```

File: zh_CN/admin/cli.md [hunk 2]
```diff
@@ -337,6 +337,29 @@ t/1 -> emqx@127.0.0.1
 如果集群中有大量的主题订阅，`list` 指令可能会比较耗时且耗资源。
 :::
 
+## exclusive topics
```
Comment (Meggielqk) [inline zh_CN/admin/cli.md]:
```suggestion
## exclusive
```

File: zh_CN/admin/cli.md [hunk 3]
```diff
@@ -337,6 +337,29 @@ t/1 -> emqx@127.0.0.1
 如果集群中有大量的主题订阅，`list` 指令可能会比较耗时且耗资源。
 :::
 
+## exclusive
+
+此命令用于查看当前系统中所有排它订阅的主题。
```
Comment (Meggielqk) [inline zh_CN/admin/cli.md]:
```suggestion
此命令用于查看当前系统中所有排它订阅的主题或删除一个排它订阅主题。
```

**Review by Meggielqk — APPROVED** [review]

**Pull request merged**